### ISSUE TITLE
CookieConsent.com

### ISSUE THREAD

**Issue by robsonsobral** [lekoala/silverstripe-cookieconsent#1]
Hi!

Please, do you know how is responsible for cookieconsent.com? It's strange their code isn't anywhere.

**Comment by lekoala** [comment]
@robsonsobral i have no idea ! but their script works great and i really like the implementation ;-)

**Comment by lekoala** [comment]
after a bit of digging, there is a very similar script at https://www.privacypolicies.com/cookie-consent/
although not much information on any actual company or individual running this

**Comment by robsonsobral** [comment]
> 
> 
> after a bit of digging, there is a very similar script at https://www.privacypolicies.com/cookie-consent/
> although not much information on any actual company or individual running this

I came to the same conclusion. The bad part is that its size is almost the size of 🌍! However, I didn't eliminate the localization strings yet.

The one by Osano also looks good.

**Comment by lekoala** [comment]
yes, that's the only downside :-) if i had unlimited time, i would most certainly redo it in a more modular fashion. if you manage to find a solution to extract to localization strings to separate files, let me know!
